examples/fabric-admin/device_manager/DeviceSynchronization.h

```diff
@@ -65,6 +65,7 @@ class DeviceSynchronizer : public chip::app::ReadClient::Callback
     chip::Callback::Callback<chip::OnDeviceConnected> mOnDeviceConnectedCallback;
     chip::Callback::Callback<chip::OnDeviceConnectionFailure> mOnDeviceConnectionFailureCallback;
 
+    chip::Controller::DeviceController * mController;
```

Comment:
```suggestion
    chip::Controller::DeviceController * mController = nullptr;
```

Could we add a comment about the expected lifetime of this?  We seem to set this on `StartDeviceSynchronization` however we never reset it back to nullptr. How do we guarantee the validity of this pointer? This may need to be reflected in the documentation of `StartDeviceSynchronization`

Reply:
Let me know if the most recent changes are acceptable